### DISCUSSION TITLE
Performance and Optimization Recommendations

### DISCUSSION THREAD

**Discussion by Bcompartment** [bevyengine/bevy#11723] [General]
Not every user is an expert on Bevy, so I want to open a discussion about the possibilities of optimizing the project.

1.  It`s a good idea to give the end user a basic optimization roadmap 
2.  What features give FPS boost In **large** scenes, but add overhead in **small** scenes 
3.  Basic features list that greatly reduce FPS on Low-End And Mobile Systems

If anyone has any ideas, please join the discussion.

**Comment by homebrewmellow** [comment]
most examples have insane overhead i assume. because simple scenes with just 3 rotated entities, run very bad on super low spec. For this reason i am avoiding developing with bevy and use a worse framework like ggez instead. I would use Bevy, but i don't know exactly what i need to change
